pallets/polkadex/src/tests.rs

```diff
@@ -24,7 +24,7 @@ fn setup_balances() {
     // Creates first asset to alice's account
     assert_ok!(pallet_generic_asset::Module::<Test>::create_asset(None,Some(alice),options_alice));
     let options_bob = pallet_generic_asset::AssetOptions::<u128, u64> {
-        initial_issuance: 1 * UNIT,
```

Comment:
@visiondream3 Please don't modify the initial_issuance; it will break other tests. Add new values as a new function.